plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ideMaven/MavenRunnerUtils.kt

```diff
@@ -38,8 +38,20 @@ fun runMavenCopyCommands(sourceFolder: File, buildlogBuilder: StringBuilder, log
         val transformMvnRunner = TransformMavenRunner(project)
         val mvnSettings = MavenRunner.getInstance(project).settings.clone() // clone required to avoid editing user settings
 
+        // run copy dependencies
+        val copyDependenciesRunnable = runMavenCopyDependencies(sourceFolder, buildlogBuilder, mvnSettings, transformMvnRunner, destinationDir, logger)
+        copyDependenciesRunnable.await()
+        buildlogBuilder.appendLine(copyDependenciesRunnable.getOutput())
+        if (copyDependenciesRunnable.isComplete() == 0) {
+            val successMsg = "IntelliJ bundled Maven copy-dependencies executed successfully"
```

Comment:
localize?

Reply:
Since this was auto-merged I can fix this in my next PR